hs_core/views/resource_rest_api.py

```diff
@@ -43,6 +43,7 @@ def resourceToResourceListItem(self, r):
         science_metadata_url = site_url + reverse('get_update_science_metadata', args=[r.short_id])
         resource_map_url = site_url + reverse('get_resource_map', args=[r.short_id])
         resource_url = site_url + r.get_absolute_url()
+        coverages = [{"type": v['type'], "value": json.loads(v['_value'])} for v in r.metadata.coverages.values()]
```

Comment:
@aphelionz Can you use ```v['value']``` instead of ```json.loads(v['_value'])```? ```value``` is defined as a property of ```Coverage``` class that returns ```json.loads(self._value)```, so I think using ```value``` property is better if possible.